audobject/core/api.py

```diff
@@ -32,6 +35,7 @@ def from_dict(  # noqa: D417
         auto_install: install missing packages needed to create the object
         override_args: override arguments in ``d`` or
             default values of hidden arguments
+        kwargs: arguments to add to override
```

Comment:
`kwargs` is deprecated and should no longer be used, that was the reason why we have not documented it before.

Reply:
I have seen the deprecation: the only purpose was to silence the `pre-commit`  hook complaining about ruff errors.  Alternatively, I could add something like a  `noqa ` .

Reply:
I have reverted these changes.